src/utils/backend.js

```diff
@@ -1,8 +1,9 @@
 import { BACKEND_URL } from '../config/constants';
 import { wrapTry } from './util';
 
-const backendFetch = (path, ...args) => wrapTry(fetch(`${BACKEND_URL}/${path}`, ...args)
-  .catch((err) => console.error(err)));
+const backendFetch = (path, ...args) => wrapTry(
+  fetch(`${BACKEND_URL}/${path}`, ...args).catch((err) => console.error(err)),
+);
```

Comment:
Would be so nice to remove `console.error(err)` from here, but it is out of the scope of this PR